2024-09-09 07:47:21,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-09 07:47:21,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-09 07:47:21,261 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-09 07:47:21,261 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-09 07:47:21,265 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-09 07:47:21,267 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-09 07:47:21,268 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-09 07:47:21,269 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-09 07:47:21,269 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-09 07:47:21,270 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_074721-rsj8wcym/logs/debug.log
2024-09-09 07:47:21,271 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_074721-rsj8wcym/logs/debug-internal.log
2024-09-09 07:47:21,272 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-09 07:47:21,273 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-09 07:47:21,273 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-09 07:47:21,274 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-09 07:47:21,281 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-09 07:47:21,286 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-09 07:47:21,318 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-09 07:47:21,319 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-09 07:47:21,320 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-09 07:47:21,368 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-09 07:47:22,361 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-09 07:47:23,270 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-09 07:48:51,719 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:48:51,720 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:51:58,215 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-09 07:51:58,256 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-09 07:51:58,257 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-09 07:51:58,258 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-09 07:51:58,258 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-09 07:51:58,259 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-09 07:51:58,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-09 07:51:58,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-09 07:51:58,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-09 07:51:58,261 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-09 07:51:58,265 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_075158-3xel5nak/logs/debug.log
2024-09-09 07:51:58,267 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_075158-3xel5nak/logs/debug-internal.log
2024-09-09 07:51:58,267 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-09 07:51:58,269 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-09 07:51:58,269 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-09 07:51:58,269 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-09 07:51:58,271 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-09 07:51:58,272 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-09 07:51:58,276 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-09 07:51:58,277 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-09 07:51:58,277 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-09 07:51:58,327 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-09 07:53:28,701 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:53:28,711 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
